pyuvdata/tests/test_calfits.py

```diff
@@ -52,13 +41,9 @@ def test_readwriteread_delays():
     uv_out = UVCal()
     testfile = os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvc.fits')
     write_file = os.path.join(DATA_PATH, 'test/outtest_firstcal.fits')
-    read_status = uvtest.checkWarnings(uv_in.read_calfits, [testfile],
-                                       nwarnings=0)
+    uvtest.checkWarnings(uv_in.read_calfits, [testfile], nwarnings=0)
```

Comment:
I thought about removing this entirely, my thinking was similar to yours. I'm happy to do whatever you think is best.